components/director/run.sh

```diff
@@ -81,24 +89,34 @@ cat ${ROOT_PATH}/../schema-migrator/seeds/director/*.sql | \
     docker exec -i ${POSTGRES_CONTAINER} psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}"
 
 if [[  ${SKIP_APP_START} ]]; then
-        echo -e "${GREEN}Skipping starting application${NC}"
-        while true
-        do
-            sleep 1
-        done
+    echo -e "${GREEN}Skipping starting application${NC}"
```

Comment:
maybe similarly to how this exists, it makes sense to "skip cleaning up the database and reuse it on next debug session" - so that we dont have to recreate all stuff every time